modin/test/backends/pandas/test_internals.py

```diff
@@ -11,7 +11,11 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+import pytest
 import modin.pandas as pd
+from modin.pandas.test.utils import create_test_dfs
+
+pd.DEFAULT_NPARTITIONS = 4
```

Comment:
Why is this equal to 4?

Reply:
This is the default value for all testing files to make tests reproducible [[1]](https://github.com/modin-project/modin/blob/ce2bea8867739ea792cdda0dce8ad50e366ebae1/modin/pandas/test/test_concat.py#L22) [[2]](https://github.com/modin-project/modin/blob/ce2bea8867739ea792cdda0dce8ad50e366ebae1/modin/pandas/test/dataframe/test_default.py#L42) [[3]](https://github.com/modin-project/modin/blob/ce2bea8867739ea792cdda0dce8ad50e366ebae1/modin/pandas/test/test_series.py#L71) [[4]](https://github.com/modin-project/modin/blob/ce2bea8867739ea792cdda0dce8ad50e366ebae1/modin/pandas/test/test_groupby.py#L30) ...

This value was mismatched in `test_internals.py`, that's why we see these changes